CHANGELOG.md

```diff
@@ -1,5 +1,352 @@
+- Replace hsr_core dependency for react profiling with hz_tracing dependency ([0f55ef7754](https://github.com/facebook/react-native/commit/0f55ef7754109c68a0bd10d958d8c81ead5e42e2) by [@metaadrianstone](https://github.com/metaadrianstone))
+- Translation auto-update for batch 2/58 on master ([89bcbed1a5](https://github.com/facebook/react-native/commit/89bcbed1a54a3c58c115d3d9ac78d211e561632a) by Intl Scheduler)
+- Translation auto-update for batch 0/58 on master ([e6fc1ecb6f](https://github.com/facebook/react-native/commit/e6fc1ecb6f68b5a59828a274c186ae98fa094d92) by Intl Scheduler)
+- Fix CQS signal. Id] 31464309 -- modernize-concat-nested-namespaces in xplat/js/react-native-github/packages/react-native/ReactAndroid/src/main/jni/react/runtime/jni ([725d9683b3](https://github.com/facebook/react-native/commit/725d9683b3affa1eda8e4a970ebff9cf0d86ad63) by generatedunixname89002005287564)
+- Fix CQS signal. Id] 22447575 -- modernize-use-nullptr in xplat/js/react-native-github/packages/react-native/ReactAndroid/src/main/jni/react/jni ([0437c4071b](https://github.com/facebook/react-native/commit/0437c4071b1cebbcbdd6c52ef2de6ceb5e2b73b2) by generatedunixname89002005287564)
+
+#### iOS Unknown
+
+- Run swift-format OrderedImports rule across fbsource [58] ([aa98b8589b](https://github.com/facebook/react-native/commit/aa98b8589bcbd4823774b5fd658372ede0d134f5) by [@ebgraham](https://github.com/ebgraham))
+- Apply SWIFTFORMAT to fbsource ([634f06688c](https://github.com/facebook/react-native/commit/634f06688cdbebcfbdc4a76f4d78c976f6209ed4) by [@darichey](https://github.com/darichey))
+- RCTDeviceInfo: fix crash due to failure to get AccessibilityManager ([ac23323da1](https://github.com/facebook/react-native/commit/ac23323da1bda2ce271797aa58dd74ffb0a5992f) by Adam Ernst)
+
+#### Failed to parse
+
+- Fix bug: unstable_hasComponent(*) = true for unregistered components for n > 1th call. ([f4d99d6a23](https://github.com/facebook/react-native/commit/f4d99d6a23a08c2ef2c2fff78dc966961608683b) by [@RSNara](https://github.com/RSNara))
+- Collections DSL functions for Kotlin(`buildReadableMap`, `buildReadableArray`) ([78dbbaafdd](https://github.com/facebook/react-native/commit/78dbbaafdd4a732f6cbb1525816b38a51adcf1cf) by [@l2hyunwoo](https://github.com/l2hyunwoo))
+- Fix changelog entry for v0.79.0-rc.2 ([ff518d799b](https://github.com/facebook/react-native/commit/ff518d799b7cc25beae2484a236ad7f16c2e629c) by [@fabriziocucci](https://github.com/fabriziocucci))
+- Add changelog entry for v0.79.0-rc.2 ([2259db6d4b](https://github.com/facebook/react-native/commit/2259db6d4bd30eda91aa952e40e98bd27f758fb6) by [@fabriziocucci](https://github.com/fabriziocucci))
+- Remove (more) unused variables from AtWork ([486d38c024](https://github.com/facebook/react-native/commit/486d38c0242989e0d22b0d9b13dd597a29014c0f) by [@ellishg](https://github.com/ellishg))
```

Comment:
We'll need to find a better place for these and/or remove some of these entries (e.g. Fix changelog entry for v0.79.0-rc.2 can be removed).